src/output_panel.c

```diff
@@ -526,129 +537,223 @@ static gboolean period_draw_event(GtkWidget *widget, cairo_t *c, struct output_p
 	return FALSE;
 }
 
+/* Return scale value from button.  A scale of 1.0 means the beat length, while
+ * a scale of 0.10 would be one tenth of a beat length.  */
+static double get_beatscale(GtkScaleButton *b)
+{
+	const double σ = log(scale_max/scale_min) / (zoom_max - zoom_min);
```

Comment:
special character "sigma" and "omega" break my compiler, and should be replaced E.g.:

```
	const double sigma = log(scale_max/scale_min) / (zoom_max - zoom_min);
	const double omega = pow(scale_max/scale_min, -1.0/(zoom_max - zoom_min)) / scale_max;

	const double zoom = gtk_scale_button_get_value(b);
	const double scale = omega * exp(sigma*zoom);
```